packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -670,7 +670,7 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
     if (result.passed)
       return true;
 
-    generateFailureOutput(result, golden, basedir);
+    await generateFailureOutput(result, golden, basedir);
```

Comment:
You can use TestAsyncUtild.giard for the method being called here and it will throw if you fail to await

Reply:
Thanks @dnfield!